src/main/resources/data.sql

```diff
@@ -0,0 +1,2 @@
+insert into person (id, first_name, last_name, gender, phone_number, date_of_birth, bio, privacy) values
+    ('2C20B8C0469311EBB3780242AC130001', 'Jovan', 'Svorcan', 0, '1111111', '1998-03-31 23:50:00', 'bio1', 0);
```

Comment:
Is this a valid UUID?

Reply:
Yes, it has been proven valid when used in another project